logic/peer.js

```diff
@@ -146,7 +149,6 @@ Peer.prototype.parseInt = function (integer, fallback) {
  */
 Peer.prototype.applyHeaders = function (headers) {
 	headers = headers || {};
-	headers = this.normalize(headers);
 	this.update(headers);
```

Comment:
ok I agree, it might cause changed behaviour in case when we are relying on result returned by `acceptHeaders` function. I will revert it